### DISCUSSION TITLE
如何解决Couldnt find main entry point for PAGeporter aex

### DISCUSSION THREAD

**Discussion by MaShangren** [Tencent/libpag#1984] [Q&A]
After Efectsia it: Couldnt find main entry point for PAGeporter aex  如何解决安装上跳出的这个问题？

**Comment by jinwuwu001** [comment]
请问该问题在最新版本中是否还存在，若存在包括PAGViewer版本、AE版本、系统版本等

**Comment by jinwuwu001** [comment]
该问题长时间未更新，暂时先关闭，后续有补充可以重新打开
